gm4_mob_curing/data/gm4_mob_curing/functions/cure/liab/check.mcfunction

```diff
@@ -0,0 +1,5 @@
+# Checks Lightning in a Bottle-specific entity reverting
+# run from main
+
+execute at @e[type=item,nbt={Item:{id:"minecraft:totem_of_undying"}},limit=1,sort=random] as @e[type=illusioner,distance=..1.5,limit=1,nbt={ActiveEffects:[{Id:1b}]}] unless data entity @s NoAI at @s run function gm4_mob_curing:cure/liab/illusioner
+execute at @e[type=item,nbt={Item:{id:"minecraft:crossbow"}},limit=1,sort=random] as @e[type=evoker,distance=..1.5,limit=1,nbt={ActiveEffects:[{Id:2b,Amplifier:3b}]}] unless data entity @s NoAI at @s run function gm4_mob_curing:cure/liab/evoker
```

Comment:
These could also be moved into the aforementioned single function file that uses `@s` to analyze the item further.